internal/storage/db/mysql/mysql.go

```diff
@@ -25,7 +25,10 @@ import (
 	"github.com/cerbos/cerbos/internal/storage/db/internal"
 )
 
-const DriverName = "mysql"
+const (
+	DriverName      = "mysql"
+	urlToSchemaDocs = "https://docs.cerbos.dev/cerbos/latest/configuration/storage.html#_database_object_definitions_2"
```

Comment:
Let's add an anchor ID to the documentation to make this link nicer rather than saying "...object_definitions_2"